lib/resty/rediscluster.lua

```diff
@@ -28,6 +29,9 @@ local DEFAULT_KEEPALIVE_CONS = 1000
 local DEFAULT_CONNECTION_TIMEOUT = 1000
 local DEFAULT_SEND_TIMEOUT = 1000
 local DEFAULT_READ_TIMEOUT = 1000
+local CONSTANT_TWO = 2
+local REDIS_VERSION_70 = 7.0
```

Comment:
Remove these constants, they do not contribute to readability.

Reply:
Done.